prime-router/metadata/hl7_mapping/resources/Encounter/PV1.yml

```diff
@@ -0,0 +1,160 @@
+elements:
+
+  - name: pv1-set-id
+    value: [ '"1"' ]
+    hl7Spec: [ '%{hl7PV1Field}-1' ]
+
+  - name: pv1-patient-class
+    resource: '%resource.status.extension(%`rsext-pv1-2-patient-class`).value'
+    schema: ../../datatypes/codeableConcept/CWE
+    constants:
+      cweFieldPath: '%{hl7PV1Field}-2'
+
+  - name: pv1-assigned-patient-location
+    resource: '%resource.location.where(extension(%`rsext-hl7-use`).value = "assigned-patient-location").location.resolve()'
+    schema: ../Location/PL/PL
+    constants:
+      hl7PLField: '%{hl7PV1Field}-3'
+
+  - name: pv1-admission-type
+    resource: '%resource.type'
+    schema: ../../datatypes/codeableConcept/CWE
+    constants:
+      cweFieldPath: '%{hl7PV1Field}-4'
+
+  - name: pv1-preadmit-number
+    resource: '%resource.hospitalization.preAdmissionIdentifier'
+    schema: ../../datatypes/identifier/CX
+    constants:
+      cxFieldPath: '%{hl7PV1Field}-5'
+
+  - name: pv1-assigned-patient-location
+    resource: '%resource.location.where(extension(%`rsext-hl7-use`).value = "prior-patient-location").location.resolve()'
+    schema: ../Location/PL/PL
+    constants:
+      hl7PLField: '%{hl7PV1Field}-6'
+
+  - name: pv1-attending-doctor
+    resource: '%resource.participant.where(individual.resolve().extension(%`rsext-hl7v2Name`).value = "pv1-7-attending-doctor").individual.resolve()'
```

Comment:
What do you think of checking for `participant.type.coding.code = "ATND"` instead of using `hl7v2Name` here and in other cases where the `type.coding.code` seems to reliably indicate which field the participant belongs in? It wouldn't work for every case, for instance for `REF` since it can be PV1.8 or PV2.13, but could reduce the burden for eventual fhir senders. I don't feel strongly about it, so consider it optional, but curious what you think.

Reply:
I had gone that direction, but wanted to a) keep consistent and b) realized that since there could be any number of participants with the same type,  it was easiest just to be extremely deterministic.